backend/src/main/java/pl/krkteam/battleships/ships/placing/validation/ShipController.java

```diff
@@ -83,7 +83,8 @@ private Game getGameFromRoom(String roomName) {
 
     private void resetGameBoardIfValidationFailed(PlacingValidationResultDTO placingValidationResultDTO,
                                                   GameBoard playerGameBoard) {
-        if (placingValidationResultDTO.getResult().equals(PlacingValidationResultDTO.Result.WRONG)) {
+        if (placingValidationResultDTO.getResult()
```

Comment:
train, extract to separate method

Reply:
doesn't make any sense to extract it to separate method